### PR TITLE
docs: submit to app hub guide

### PR DESCRIPTION
**Pull request by deboragaleano**
* This is a rewrite of the guide that lived on the [DHIS2 main site](https://www.dhis2.org/upload-app-to-dhis2-app-store)
* Included some of the guidelines drafted by @cooper-joe
* This guide was created as a blog to allow for comments and feedback from the dev community 

TODO

- [X] Add link to the **submission guidelines** document
- [X] Review the FAQ section

### CODE REVIEW

**Comment by amcgee** [comment]
Nice @deboragaleano !  Do you want to address the TODOs here in this PR or merge this and address them later?

**Comment by amcgee** [comment]
(from live discussion) we'll address these issues in this PR, so I'm changing this to a draft 🥇

File: _posts/2021-01-22-submit-apphub.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+##### General
+
+-   **App name**: Be clear and specific. Try to capture the core functionality of your app.
+-   **App Description**: What's the purpose of your app? What does the app allow a user to do? What are the main functionalities? Any technical requirements?
+-   **Source Code URL**: Please provide a link to a public repository. This will allow the DHIS2 core team to quickly review your app.
+
+##### Version
+
+-   **Version**: Version of your app. [Semantic versioning](https://semver.org/){:target="\_blank"} recommended (i.e. 1.0.0)
```
Comment (amcgee) [inline _posts/2021-01-22-submit-apphub.md:37]:
```suggestion
-   **Version**: The version number of your app. [Semantic versioning](https://semver.org/){:target="\_blank"} is recommended (i.e. 1.0.0)
```

File: _posts/2021-01-22-submit-apphub.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
+---
+title: Submit an app to the DHIS2 App Hub
```
Comment (amcgee) [inline _posts/2021-01-22-submit-apphub.md:2]:
```suggestion
title: How to submit an app to the DHIS2 App Hub
```

**Review by amcgee — APPROVED** [review]
Looks great @deboragaleano !  I made one small suggestion.  It looks like some of the tag pages were already created in #47 ...?  Not sure why they still show up as new files...

Bigger question - do you think this should be a blog post or a guide?  Or maybe both, and it can be linked from the guides page?

**Comment by deboragaleano** [comment]
Thanks for reviewing this guide @amcgee ! I'm not sure why the same tags (`authentication` and `troubleshooting`) are being created, actually 🤔  

As for this being a blog or a guide, it's definitely a guide. I just thought that the `post` layout instead of the `page` one was better - to allow blog post comments and feedback. I think I'll move this to the `guides` folder instead, use a `post` layout and then yes, link it from the guides page. What do you think?

> Looks great @deboragaleano ! I made two small suggestions. It looks like some of the tag pages were already created in #47 ...? Not sure why they still show up as new files...
> 
> Bigger question - do you think this should be a blog post or a guide? Or maybe both, and it can be linked from the guides page?

**Comment by deboragaleano** [comment]
Sorry for overcomplicating things! I decided to move this guide to the guides folder and kept the formatting like the rest (as a page).

**Review by amcgee — APPROVED** [review]
No problem, looks good!  Feel free to merge when it's ready :-)

I think it could also be a blog post, or at least we could announce it on the community of practice - no rush to do either though, let's go ahead and publish now 👍

**Pull request merged**